internal/cli/atlas/clusters/create.go

```diff
@@ -79,20 +80,13 @@ func (opts *CreateOpts) Run() error {
 	return opts.Print(r)
 }
 
-func (opts *CreateOpts) newCluster() (*atlas.Cluster, error) {
-	cluster := new(atlas.Cluster)
+func (opts *CreateOpts) newCluster() (*atlas.AdvancedCluster, error) {
+	cluster := new(atlas.AdvancedCluster)
 	if opts.filename != "" {
 		if err := file.Load(opts.fs, opts.filename, cluster); err != nil {
 			return nil, err
 		}
-		// There can only be one
-		if cluster.ReplicationSpecs != nil {
-			cluster.ReplicationSpec = nil
-		}
 		// This can't be sent
-		cluster.MongoURI = ""
-		cluster.MongoURIWithOptions = ""
-		cluster.MongoURIUpdated = ""
```

Comment:
[q] are you deleting because they are no longer needed? this was there so if you did `cluster get > file.json` you could do `cluster create <newName> -f file.json` if that still works 👍

Reply:
ah okay, some of them are no longer needed but this scenario doesn’t work because I have to remove the ids as I am doing with update. Working on it

Reply:
With the latest changes:
```
/bin/mongocli atlas cluster get Cluster2 -o json > fileTest.json
➜  ~/workspace/mongoCLI git:(CLOUDP-74979) ✗ ./bin/mongocli atlas cluster create ClusterTest2 --file fileTest.json
Deploying cluster ClusterTest2.
```